.github/workflows/ci.yaml

```diff
@@ -1,7 +1,7 @@
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
I believe GitHub supports `$default-branch` as an identifier we could use here as well, which may be nice